core/src/scene/asset.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <memory>
+#include <map>
+
+namespace Tangram {
+
+class Platform;
+struct ZipHandle;
+
+class Asset {
+    public:
```

Comment:
The class definition is indented 4 spaces more than it should be, probably an IDE trying to "help".